config/jest/test.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     },
     collectCoverageFrom: [
         "packages/**/*.js",
+        "shared-unpackaged/**/*.js",
```

Comment:
Turns out we weren't collecting coverage data for the unpackaged code that gets shared in several places (the sharing is done this way because we didn't want to expose these pieces as part of a public API).